### PR TITLE
Add new Performance Analyzer metrics and fix table formatting

### PR DESCRIPTION
**Pull request by hdhalter**
### Description
The table had some formatting issues, so I fixed the table formatting and added the new metrics. This takes the place of:  https://github.com/opensearch-project/documentation-website/pull/5114

### Issues Resolved
n/a
https://github.com/opensearch-project/performance-analyzer/pull/483

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by ruai0511** [comment]
Thank you for re-formatting! This makes sense. I added two rows to the Dimension Reference table in the old pull request which aren't present here.

**Comment by hdhalter** [comment]
Thanks, @ruai0511 ! I added the two items in the Dimensions section. I'm still working out some of the kinks in the formatting, and should have another version up soon. If you have any information about the relationship between the metrics and dimensions, that would be helpful. I tried to provide some context in the opening paragraph, but not sure if I got it right. Thanks, again.

**Comment by hdhalter** [comment]
I have a few questions for dev/PM: 
- Can you please explain the relationship between the metrics and dimensions? I would like to provide some context as the intro to the topic and as the intro to the dimensions table at the bottom.
- What is WIP? Can we replace it with something meaningful?
- Why is one of the tables "Dimensions: N/A"?  What does that mean?
Thanks!

**Comment by hdhalter** [comment]
I removed the Log Name column until we can get the long names built in the UI. @jimishs - please approve the updates. Thanks!

File: _monitoring-your-cluster/pa/reference.md
Comment (Naarcha-AWS) [inline _monitoring-your-cluster/pa/reference.md:12]:
```suggestion
OpenSearch provides a number of metrics to help you evaluate performance. The following tables describe the metrics available, grouped by the dimensions that are most relevant for that metric. All metrics support the `avg`, `sum`, `min`, and `max` aggregations, although certain metrics measure one thing, making the choice of aggregation irrelevant.
```

File: _monitoring-your-cluster/pa/reference.md
Comment (Naarcha-AWS) [inline _monitoring-your-cluster/pa/reference.md:128]:
```suggestion
      <td>Average time (seconds) that the associated thread(s) waited to enter or reenter a monitor in the `WAITING` or `TIMED_WAITING` state.
```

Comment (Naarcha-AWS) [reply _monitoring-your-cluster/pa/reference.md:128]: Might have to use `<code>` instead of the ` since this is formatted in HTML.

File: _monitoring-your-cluster/pa/reference.md
Comment (Naarcha-AWS) [inline _monitoring-your-cluster/pa/reference.md:134]:
```suggestion
      <td>The total number of times that the associated thread(s) waited to enter or reenter a monitor (that is, the number of times a thread has been in the <code>WAITING</code> or <code>TIMED_WAITING</code> state).
```

File: _monitoring-your-cluster/pa/reference.md
Comment (Naarcha-AWS) [inline _monitoring-your-cluster/pa/reference.md:600]:
```suggestion
     <td>The current number of pending tasks in the cluster state update thread. Each node has a cluster state update thread that submits cluster state update tasks, such as create index, update mapping, allocate shard, and, fail shard.
```

File: _monitoring-your-cluster/pa/reference.md
Comment (Naarcha-AWS) [inline _monitoring-your-cluster/pa/reference.md:816]:
```suggestion
     <td>The state of each shard, for example, `STARTED`, `UNASSIGNED`,  and `RELOCATING`.
```

File: _monitoring-your-cluster/pa/reference.md
Comment (Naarcha-AWS) [inline _monitoring-your-cluster/pa/reference.md:969]:
```suggestion
     <td>The number of tasks marked for cancellation because of excessive CPU usage since the node last restarted on the search task level.
```

File: _monitoring-your-cluster/pa/reference.md
Comment (natebower) [inline _monitoring-your-cluster/pa/reference.md:718]: This is a class name `AdmissionController`. "The total number of rejections performed by the `AdmissionController`"?

File: _monitoring-your-cluster/pa/reference.md
Comment (natebower) [inline _monitoring-your-cluster/pa/reference.md:724]:
```suggestion
    <td>The current value for Controller of Admission Control.
```

Comment (natebower) [inline _monitoring-your-cluster/pa/reference.md:724]: "The current value of the `AdmissionController`"?

File: _monitoring-your-cluster/pa/reference.md
Comment (natebower) [inline _monitoring-your-cluster/pa/reference.md:730]:
```suggestion
    <td>The threshold value for Controller of Admission Control.
```

Comment (natebower) [inline _monitoring-your-cluster/pa/reference.md:730]: "The threshold value of the `AdmissionController`"?

File: _monitoring-your-cluster/pa/reference.md
Comment (natebower) [inline _monitoring-your-cluster/pa/reference.md:749]:
```suggestion
      <td>The number of throttled pending tasks on which the data node is actively performing retries. It is an absolute metric at that point in time.
```

Comment (natebower) [inline _monitoring-your-cluster/pa/reference.md:749]: The second sentence here is a little awkward/unclear. Can we simplify/clarify? Something like "This is an absolute metric when..."

File: _monitoring-your-cluster/pa/reference.md
Comment (natebower) [inline _monitoring-your-cluster/pa/reference.md:755]:
```suggestion
      <td>The sum of the total pending tasks that were throttled by the cluster manager node. This is a cumulative metric, so make sure to check the max aggregation.
```

File: _monitoring-your-cluster/pa/reference.md
Comment (natebower) [inline _monitoring-your-cluster/pa/reference.md:761]: Is "N/A" a placeholder here?

File: _monitoring-your-cluster/pa/reference.md
Comment (natebower) [inline _monitoring-your-cluster/pa/reference.md:773]:
```suggestion
     <td>A number that increases monotonically with every cluster manager election.
```

File: _monitoring-your-cluster/pa/reference.md
Comment (natebower) [inline _monitoring-your-cluster/pa/reference.md:779]:
```suggestion
     <td>The amount of time taken by the quorum of nodes to publish the new cluster state. This metric is available for the current cluster manager.
```

File: _monitoring-your-cluster/pa/reference.md
Comment (natebower) [inline _monitoring-your-cluster/pa/reference.md:785]: Should "the publish" be here? Should this read "The number of times that the new cluster state action failed on the cluster manager node"?

File: _monitoring-your-cluster/pa/reference.md
Comment (natebower) [inline _monitoring-your-cluster/pa/reference.md:791]:
```suggestion
     <td>The amount of time taken by each node for the apply cluster state sent by the cluster manager.
```

Comment (natebower) [inline _monitoring-your-cluster/pa/reference.md:791]: Should "action" follow "state"?

File: _monitoring-your-cluster/pa/reference.md
Comment (natebower) [inline _monitoring-your-cluster/pa/reference.md:816]:
```suggestion
     <td>The state of each shard, for example, `STARTED`, `UNASSIGNED`, or `RELOCATING`.
```

File: _monitoring-your-cluster/pa/reference.md
Comment (natebower) [inline _monitoring-your-cluster/pa/reference.md:822]: I'm assuming TBD should be replaced here.

File: _monitoring-your-cluster/pa/reference.md
Comment (natebower) [inline _monitoring-your-cluster/pa/reference.md:879]:
```suggestion
     <td>The number of times that the number of tasks eligible for cancellation exceeded the set cancellation threshold on the shard task.
```

File: _monitoring-your-cluster/pa/reference.md
Comment (natebower) [inline _monitoring-your-cluster/pa/reference.md:897]:
```suggestion
     <td> The rolling average heap usage for the n most recent tasks on the shard task. The default value for n is 100.
```

File: _monitoring-your-cluster/pa/reference.md
Comment (natebower) [inline _monitoring-your-cluster/pa/reference.md:903]:
```suggestion
     <td>The number of tasks marked for cancellation because of excessive CPU usage since the node last restarted on the shard task.
```

File: _monitoring-your-cluster/pa/reference.md
Comment (natebower) [inline _monitoring-your-cluster/pa/reference.md:909]: Should "time" be "usage"?

File: _monitoring-your-cluster/pa/reference.md
Comment (natebower) [inline _monitoring-your-cluster/pa/reference.md:915]: Should "time" be "usage"?

File: _monitoring-your-cluster/pa/reference.md
Comment (natebower) [inline _monitoring-your-cluster/pa/reference.md:945]:
```suggestion
     <td>The number of times that the number of tasks eligible for cancellation exceeded the set cancellation threshold on the search task level.
```

File: _monitoring-your-cluster/pa/reference.md
Comment (natebower) [inline _monitoring-your-cluster/pa/reference.md:963]:
```suggestion
     <td> The rolling average heap usage for the n most recent tasks on the search task level. The default value for n is 10.
```

File: _monitoring-your-cluster/pa/reference.md
Comment (natebower) [inline _monitoring-your-cluster/pa/reference.md:975]: Should "time" be "usage"?

File: _monitoring-your-cluster/pa/reference.md
Comment (natebower) [inline _monitoring-your-cluster/pa/reference.md:981]: Should "time" be "usage"?

File: _monitoring-your-cluster/pa/reference.md
Comment (natebower) [inline _monitoring-your-cluster/pa/reference.md:1012]:
```suggestion
| ShardRole            | The shard role, for example, `primary` or `replica`.                            |
```

File: _monitoring-your-cluster/pa/reference.md
Comment (natebower) [inline _monitoring-your-cluster/pa/reference.md:1019]:
```suggestion
| Direction            | The direction, for example, `in` or `out`.                                    |
```

File: _monitoring-your-cluster/pa/reference.md
Comment (natebower) [inline _monitoring-your-cluster/pa/reference.md:1023]:
```suggestion
| ClusterManagerTaskPriority   | The priority of the task, for example, `URGENT`. OpenSearch executes higher-priority tasks before lower-priority ones, regardless of `insert_order`. |
```

File: _monitoring-your-cluster/pa/reference.md
Comment (natebower) [inline _monitoring-your-cluster/pa/reference.md:1009]: "of" instead of "for"?

File: _monitoring-your-cluster/pa/reference.md
Comment (natebower) [inline _monitoring-your-cluster/pa/reference.md:1027]: "of" instead of "for"?

**Review by natebower — CHANGES_REQUESTED** [review]
@hdhalter Please see my comments and changes and let me know if you have any questions. It looks like there are still placeholders/unfinished elements in places, I suspect that some terms should be in code font that are not currently, and I think that some of the phrasing may be inaccurate, so please get a doc review before approval. Thanks!

File: _monitoring-your-cluster/pa/reference.md
Comment (khushbr) [inline _monitoring-your-cluster/pa/reference.md:12]: Wondering if we should mention `Performance Analyzer` here. `OpenSearch Performance Analyzer plugin provides a number of metrics to ...`, The Performance Analyzer plugin can be turned on and off, so it is important to call out that metrics are coming from the plugin

File: _monitoring-your-cluster/pa/reference.md
Comment (hdhalter) [reply _monitoring-your-cluster/pa/reference.md:761]: That's correct, but I added a description to explain what that means.

File: _monitoring-your-cluster/pa/reference.md
Comment (hdhalter) [reply _monitoring-your-cluster/pa/reference.md]: Updated, thanks! And confirmed that `TIMED` is correct.

File: _monitoring-your-cluster/pa/reference.md
Comment (natebower) [inline _monitoring-your-cluster/pa/reference.md:762]:
```suggestion
The following metrics are relevant to the cluster as a whole and do not require specific dimensions.
```

**Review by natebower — APPROVED** [review]
LGTM

**Comment by hdhalter** [comment]
I removed all the new content. @khushbr  , please approve. Thanks!

**Comment by ruai0511** [comment]
@hdhalter I made another [commit](https://github.com/opensearch-project/documentation-website/pull/5114/commits/87c55b969fb5eb4f9688d89fed4fda17e979edd3) based on the reviews on my pull [request](https://github.com/opensearch-project/documentation-website/pull/5114) (I don't think I can commit to your PR since I don't have write permission.)

**Comment by hdhalter** [comment]
> @hdhalter I made another [commit](https://github.com/opensearch-project/documentation-website/pull/5114/commits/87c55b969fb5eb4f9688d89fed4fda17e979edd3) based on the reviews on my pull [request](https://github.com/opensearch-project/documentation-website/pull/5114) (I don't think I can commit to your PR since I don't have write permission.)

I saw that, but @khushbr said the search backpressure metrics were not going in 2.11 now. I will make sure your input is reflected in the content when it gets delivered.

**Comment by ruai0511** [comment]
> > @hdhalter I made another [commit](https://github.com/opensearch-project/documentation-website/pull/5114/commits/87c55b969fb5eb4f9688d89fed4fda17e979edd3) based on the reviews on my pull [request](https://github.com/opensearch-project/documentation-website/pull/5114) (I don't think I can commit to your PR since I don't have write permission.)
> 
> I saw that, but @khushbr said the search backpressure metrics were not going in 2.11 now. I will make sure your input is reflected in the content when it gets delivered.

Ah Sure. Thanks!

**Review by khushbr — APPROVED** [review]

**Comment by hdhalter** [comment]
Merging this without search backpressure metrics (which is targeting 2.12), since it fixes the formatting problems and has been edited. I've remove the 2.11 label.

**Pull request merged**
